### PR TITLE
Changes in Simbrief import, ND display, MCDU, PERFTO, CLR button

### PR DESCRIPTION
**Pull request by inuyaksa**
Adding some features about real FMS (Honeywell), I check on manual and yt videos.
Implement import of pseudo waypoints from Simbrief and speed/alt computed data.
Thanks for your great work to port A320 on Flightgear, I hope you enjoy my contribution.

### Description of Changes
- long pressing CLR button (keyboard/MCDU panel) clears scratchpad (not works on MCDU dialog)
- Simbrief import adding TOC/TOD pseudo-waypoints and speed/alt for all waypoints
- Basic manage of coRoute
- ND displays TOC/TOD and speed/alt constraints
- abort button for panel state dialog, when error occurs on state script, you can't close panel
- fix MCDU starting page, arrow color and REQ state (https://youtu.be/twwUjHXdNVU?t=57)
- PERFTO, validation on insert of V1/Vr/V2 values and type II error messages
- PERFTO, basic validation of THRRED/ACC

### Screenshots (optional)
none, sorry

### Bugs fixed (if any)
- prevent fix graphics settings dialog on false positive when custom-setting is off and quality-level >= 3
- "soft" auto-pilot disengage on roll-out phase (ILS landing)

### Checklist:
* [x] My changes follow the Contributing Guidelines.
* [x] My changes implement realistic features.
* [x] Please have a main Developer test my changes before merging.
* [x] My changes are ready for merging.

### CODE REVIEW

**Comment by legoboyvdlp** [comment]
Whew, that seems like a lot of work -- thanks!

Unfortunately I think there has been some parallel work. Specifically:

Simbrief import adding TOC/TOD pseudo-waypoints and speed/alt for all waypoints
ND displays TOC/TOD and speed/alt constraints

It would actually help if the commits had been separated out per feature, but no matter, parts can be merged manually. Or perhaps even, it can be merged entirely and then replaced later by the on-going work on this.

I'll let @mattmaring comment on this request first before proceeding further. 

Thanks again!

**Comment by legoboyvdlp** [comment]
Also, another aspect, we usually went for the Thales FMGC as well, but where differences exist they should be easy enough to modify later on.

If you intend to do any more work, would you like to join us at our development Mattermost server? :)

**Comment by inuyaksa** [comment]
Hi, yes I have tested the big work of mattmaring, how a great enhanced on this project.
I'd like to contribute, how can I join your server?

**Comment by merspieler** [comment]
for the server, please write me a pm on the forum, there I'll send you the invite link

File: Nasal/FMGC/FMGC.nas [hunk 1]
```diff
@@ -625,7 +628,7 @@ var masterFMGC = maketimer(0.2, func {
 	
 	if (FMGCInternal.phase == 1 and ((mode != "SRS" and mode != " ") or alt >= accel_agl_ft)) {
 		FMGCInternal.phase = 2;
-		systems.PNEU.pressMode.setValue("TO");
+		systems.PNEU.pressMode.setValue("TO"); # TODO - why not set to CL (climb) ??
```
Comment (legoboyvdlp) [inline Nasal/FMGC/FMGC.nas:631]: Please ignore pressurization for now. That's rather broken and needs re-done.

Comment (inuyaksa) [reply Nasal/FMGC/FMGC.nas:631]: yep. 
I've made a typo for computed_mach.
I found thales manual (rev1, I think is a lot old), I'm reading for check differences.

**Comment by mattmaring** [comment]
Thanks for working on the A320, I like these changes a lot!

As for T/C and T/D, I'm not sure that pseudo waypoints should be imported into the F-PLN on INIT REQUEST. That import triggers an automatic fuel recalculation which would immediately overwrite those values, thus they would never be displayed (fuel fields are dashed until computations are complete). The same would apply for the speed/altitude constraints, as those two are computed dynamically in the new MCDU work.

I would probably leave those two features out of the push to dev, as until the new flightplan development is pushed (most likely another couple months, at minimum), those points are static and won't update with further changes to the route (i.e. adding/removing waypoints, etc).

However, please do leave that work in as comments. I'm curious what INIT REQUEST looks like on the F-PLN before the fuel calculations are complete (~5 sec, so it's going to be hard to find a video of this). I would assume pseudo waypoints don't show up, but I've also not seen any documentation state otherwise...

**Comment by inuyaksa** [comment]
Hi, I found your work on fuel/time estimations only few days ago, it's so enormous! :D
T/C and others pseudo waypoints seems not so mandatory on F-PLN, computer display only when all data has filled.
I found a video where (LIM) has displayed only after fuel block insertion and some computation https://youtu.be/bsiLiwt_18U?t=696

**Pull request closed without merging**